build_system/src/config.rs

```diff
@@ -469,11 +463,28 @@ impl ConfigInfo {
 }
 
 fn download_gccjit(
-    url: String,
+    commit: &str,
     output_dir: &Path,
     tempfile_name: String,
     with_progress_bar: bool,
 ) -> Result<(), String> {
+    let url = if std::env::consts::OS == "linux" || std::env::consts::ARCH == "x86_64" {
+        format!("https://github.com/rust-lang/gcc/releases/download/master-{}/libgccjit.so", commit)
+    } else {
+        eprintln!(
+            r"\
```

Comment:
Is the `r` useful here?

Reply:
Quite the opposite of useful actually as a raw string is not what I want here.🤦‍♂️